app/api/routers/position.py

```diff
@@ -157,11 +169,14 @@ def __call__(
         if offset is not None:
             query = query.offset(offset)
 
-        _locked_list: list[IDXLockedPosition] = query.all()
+        _locked_list: list[tuple[IDXLockedPosition, IDXTokenInstance]] = query.all()
         locked_list = []
 
         for _locked in _locked_list:
-            locked_list.append(_locked.json())
+            _locked_data = _locked[0].json()
+            if request_query.include_token_details is True:
+                _locked_data["token"] = self.token_model.from_model(_locked[1]).__dict__
+            locked_list.append(_locked_data)
```

Comment:
- If **include_token_details=true** is queried, token_detail attribute is given.